src/System.Private.CoreLib/shared/System/Runtime/InteropServices/Marshal.cs

```diff
@@ -232,14 +231,14 @@ public static void Copy(IntPtr[] source, int startIndex, IntPtr destination, int
 
         private static unsafe void CopyToNative<T>(T[] source, int startIndex, IntPtr destination, int length)
         {
-            if (source == null)
+            if (source is null)
                 throw new ArgumentNullException(nameof(source));
             if (destination == IntPtr.Zero)
                 throw new ArgumentNullException(nameof(destination));
 
             // The rest of the argument validation is done by CopyTo
 
-            new Span<T>(source, startIndex, length).CopyTo(new Span<T>((void*)destination, length));
+            source.AsSpan(startIndex, length).CopyTo(new Span<T>((void*)destination, length));
```

Comment:
Why did you change this?

Reply:
Ah, I was prototyping an extension and forgot to disable it on my home machine. I'll revert.